lib/caching-browserify.js

```diff
@@ -71,7 +71,7 @@ module.exports = CoreObject.extend({
     rimraf.sync(this._inputStaging);
     mkdirp.sync(this._inputStaging);
 
-    symlinkOrCopy.sync(path.join(inDir, this.inFile), path.join(this._inputStaging, this.inFile));
+    symlinkOrCopy.sync(inDir + '/' + this.inFile, this._inputStaging + '/' + this.inFile);
```

Comment:
There is not point, all node FS apis already normalize. So it is quite strange to pollute user-land code with both `/` and `\\`.